src/org/labkey/test/WebDriverWrapper.java

```diff
@@ -1070,6 +1070,27 @@ protected void closeExtraWindows()
         }
     }
 
+    protected void quietlyCloseExtraWindows()
+    {
+        List<String> windows = new ArrayList<>(getDriver().getWindowHandles());
+        for (int i = 1; i < windows.size(); i++)
+        {
+            getDriver().switchTo().window(windows.get(i));
+            try
+            {
+                getDriver().close();
+            }
+            catch (UnhandledAlertException uae)
+            {
+                Optional.ofNullable(getAlertIfPresent()).ifPresent(Alert::accept);
+            }
+        }
+        if (windows.size() > 1)
+        {
+            getDriver().switchTo().window(windows.get(0));
+        }
```

Comment:
Not a fan of adding a duplicate method that differs only in logging behavior.
I would agree that the existing logging isn't especially useful though. Maybe remove the `@LogMethod` annotation from `closeExtraWindows` and have it log how many windows it closes and skip logging altogether if there aren't any windows to close.
```suggestion
        if (windows.size() > 1)
        {
            log("Closed %s tabs/windows".formatted(windows.size() - 1));
            getDriver().switchTo().window(windows.get(0));
        }
```

Reply:
This was removed due to your suggestion in the ehrModules PR.